stable/yugaware/templates/configs.yaml

```diff
@@ -176,6 +176,18 @@ data:
       location / {
         proxy_pass http://{{ eq .Values.ip_version_support "v6_only" | ternary "[::1]" "127.0.0.1" }}:9000;
       }
+{{- if .Values.nginx.db_node_proxy_enabled }}
+      location ~ "^/universes/.+/proxy/(.+):(7000|9000|9300|12000|13000)/(metrics|prometheus-metrics)$" {
+        proxy_pass "http://$1:$2/$3$is_args$args";
+        sub_filter "http://" "/proxy/";
```

Comment:
Can you explain why we need to do this filtering? I don't think the metrics endpoints should be returning any URLs.

Reply:
was implementing a more generic solution that also worked for going to the master/tserver UI before, but now that we've restricted the path to only metrics endpoints it is a moot point